test/src/userobjects/TestWeightedGapUserObject.C

```diff
@@ -0,0 +1,142 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
```

Comment:
Do we need these test objects for coverage?

Reply:
yea otherwise we don't have any mortar user object usage for framework coverage testing